llvm/lib/Target/RISCV/RISCVSchedSiFiveP400.td

```diff
@@ -45,6 +154,13 @@ defvar SiFiveP400FloatArith  = SiFiveP400FEXQ0;
 defvar SiFiveP400F2I      = SiFiveP400FEXQ0;
 def SiFiveP400FloatDiv    : ProcResource<1>;
 
+// Vector pipeline
+def SiFiveP400VEXQ0        : ProcResource<1>;
```

Comment:
`EXQ0` means `execution queue 0`? But it seems there is no second unit and all arithmetic operations are executed on this unit (not a quene?).

Reply:
There is an issue queue in front of the ALU. https://www.cnx-software.com/wp-content/uploads/2022/11/SiFive-P470-detailed-pipeline-overview.png

Reply:
So there is 1 vector pipeline? Is it OK to model VLD/VST/VDiv/VFloatDiv as standalone resources?

Reply:
There is an issue queue in front of the vector ALU (purple). Vector loads go through the load/store dispatch buffer (green). Vector divisions are non blocking to other vector ALU operations.

We think it is okay to model VLD/VST/VDiv/VFloatDiv as standalone resources.